flytekit/models/literals.py

```diff
@@ -549,7 +549,7 @@ def from_flyte_idl(cls, pb2_object):
 
 
 class StructuredDatasetMetadata(_common.FlyteIdlEntity):
-    def __init__(self, structured_dataset_type: StructuredDatasetType = None):
```

Comment:
`Optional[StructuredDatasetType]`?  does the idl translation stuff work okay if it's None?

Reply:
updated it